### PR TITLE
Fix missing make command in build_ipa.sh

### PR DESCRIPTION
**Pull request by Rozzii**

### CODE REVIEW

File: ci/scripts/image_scripts/build_ipa.sh [hunk 1]
```diff
@@ -72,6 +72,11 @@ tar --create --verbose --file="${IPA_IMAGE_TAR}" \
     "./${IPA_IMAGE_NAME}.kernel" \
     "./${IPA_IMAGE_NAME}.initramfs"
 
+wget "https://golang.org/dl/go1.17.linux-amd64.tar.gz"
+rm -rf "/usr/local/go"
+tar -C "/usr/local -xzf go1.17.linux-amd64.tar.gz"
+export PATH=$PATH:/usr/local/go/bin
+
```
Comment (fmuyassarov) [inline ci/scripts/image_scripts/build_ipa.sh:79]: this is redundant I think? or do we need golang before integration tests?

**Review by fmuyassarov — CHANGES_REQUESTED** [review]

File: ci/jobs/openstack_ipa_and_ironic_image_building.pipeline [hunk 2]
```diff
@@ -70,12 +73,21 @@ pipeline {
   }
   post{
     always{
+      script {
+        CURRENT_END_TIME = System.currentTimeMillis()
+        if ((((CURRENT_END_TIME - CURRENT_START_TIME)/1000) - TIMEOUT) > 0) {
```
Comment (fmuyassarov) [inline ci/jobs/openstack_ipa_and_ironic_image_building.pipeline:78]: `TIMEOUT` var is not declared anywhere. You either need to declare it and use that in Line55 or harcode the same value here. I would go for the first option.

**Review by fmuyassarov — CHANGES_REQUESTED** [review]

File: ci/jobs/openstack_ipa_and_ironic_image_building.pipeline (hunk 2 above)
Comment (Rozzii) [reply ci/jobs/openstack_ipa_and_ironic_image_building.pipeline:81]: Done.

**Comment by Rozzii** [comment]
I will add log collection in a separate PR.

File: ci/jobs/openstack_ipa_and_ironic_image_building.pipeline [hunk 3]
```diff
@@ -52,16 +55,19 @@ pipeline {
   stages {
     stage('Building ipa image'){
       options {
-        timeout(time: 60, unit: 'MINUTES')
+        timeout(time: 120, unit: 'MINUTES')
```
Comment (fmuyassarov) [inline ci/jobs/openstack_ipa_and_ironic_image_building.pipeline:58]:
```suggestion
        timeout(time: TIMEOUT, unit: 'MINUTES')
```

**Review by fmuyassarov — CHANGES_REQUESTED** [review]

**Review by fmuyassarov — APPROVED** [review]

**Pull request merged**